2022-01-10 15:10:15,465 INFO    MainThread:2261000 [wandb_setup.py:_flush():71] setting env: {}
2022-01-10 15:10:15,465 INFO    MainThread:2261000 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-10 15:10:15,465 INFO    MainThread:2261000 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-10 15:10:15,466 INFO    MainThread:2261000 [wandb_init.py:_log_setup():371] Logging user logs to /mnt/nfs/rdata02-users/users/makaroff/UNet_pytorch/wandb/run-20220110_151015-3gcdxydb/logs/debug.log
2022-01-10 15:10:15,466 INFO    MainThread:2261000 [wandb_init.py:_log_setup():372] Logging internal logs to /mnt/nfs/rdata02-users/users/makaroff/UNet_pytorch/wandb/run-20220110_151015-3gcdxydb/logs/debug-internal.log
2022-01-10 15:10:15,466 INFO    MainThread:2261000 [wandb_init.py:init():404] calling init triggers
2022-01-10 15:10:15,466 INFO    MainThread:2261000 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-10 15:10:15,466 INFO    MainThread:2261000 [wandb_init.py:init():460] starting backend
2022-01-10 15:10:15,466 INFO    MainThread:2261000 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-10 15:10:15,476 INFO    MainThread:2261000 [backend.py:ensure_launched():216] starting backend process...
2022-01-10 15:10:15,483 INFO    MainThread:2261000 [backend.py:ensure_launched():221] started backend process with pid: 2261019
2022-01-10 15:10:15,484 INFO    MainThread:2261000 [wandb_init.py:init():469] backend started and connected
2022-01-10 15:10:15,494 INFO    MainThread:2261000 [wandb_init.py:init():533] updated telemetry
2022-01-10 15:10:15,498 INFO    MainThread:2261000 [wandb_init.py:init():563] communicating current version
2022-01-10 15:10:16,177 INFO    MainThread:2261000 [wandb_init.py:init():568] got version response 
2022-01-10 15:10:16,177 INFO    MainThread:2261000 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-10 15:10:16,437 INFO    MainThread:2261000 [wandb_init.py:init():606] starting run threads in backend
2022-01-10 15:10:16,491 INFO    MainThread:2261000 [wandb_run.py:_console_start():1810] atexit reg
2022-01-10 15:10:16,492 INFO    MainThread:2261000 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-10 15:10:16,493 INFO    MainThread:2261000 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-10 15:10:16,496 INFO    MainThread:2261000 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-10 15:10:16,496 INFO    MainThread:2261000 [wandb_init.py:init():633] run started, returning control to user process
2022-01-10 15:12:19,671 INFO    MainThread:2261000 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-10 15:12:19,673 INFO    MainThread:2261000 [wandb_run.py:_restore():1752] restore
2022-01-10 15:12:22,033 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 463
  total_bytes: 463
}

2022-01-10 15:12:22,307 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 463
  total_bytes: 463
}

2022-01-10 15:12:22,592 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 463
  total_bytes: 14159
}

2022-01-10 15:12:22,694 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 463
  total_bytes: 14159
}

2022-01-10 15:12:22,795 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 463
  total_bytes: 14159
}

2022-01-10 15:12:22,896 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14131
  total_bytes: 14159
}

2022-01-10 15:12:22,998 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14159
  total_bytes: 14159
}

2022-01-10 15:12:23,100 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14159
  total_bytes: 14159
}

2022-01-10 15:12:23,202 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14159
  total_bytes: 14159
}

2022-01-10 15:12:23,304 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14159
  total_bytes: 14159
}

2022-01-10 15:12:23,406 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14159
  total_bytes: 14159
}

2022-01-10 15:12:23,944 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14159
  total_bytes: 14159
}

2022-01-10 15:12:24,372 INFO    MainThread:2261000 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14159
  total_bytes: 14159
}
local_info {
}

2022-01-10 15:12:25,455 INFO    MainThread:2261000 [wandb_run.py:_append_files():2180] logging synced files
